build.cmd

```diff
@@ -51,7 +51,6 @@ set __SkipConfigure=
 set __SkipMscorlibBuild=
 set __SkipNativeBuild=
 set __SkipTestBuild=
-set __DoCrossgen=
```

Comment:
It does, but every time it's immediately followed by "buildtest.cmd crossgen". So, if all this arg does is tell buildtest to do crossgen, then we don't need it for the CI. (That is, if "build.cmd skiptests; buildtest.cmd crossgen" is the same as "build.cmd docrossgen", then we can get rid of it).